.eslintrc.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
     'arrow-parens': ['error', 'as-needed', { requireForBlockBody: true }],
     'max-classes-per-file': 'off',
     'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-debugger': 'error',
```

Comment:
this is largely due to some errant `debugger` statements found in `DAO.spec.js`.

we left `no-console` as-is, as making it always `'error'` would break dev builds using `console.log()`.  our "CI-ish" deployment script catches broken AWS-mode builds.